package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "prepare": "husky install $PWD/.husky",
-    "build": "ncc build index.js -o dist --license licenses.txt",
+    "build": "ncc build src/index.js -o dist --license licenses.txt",
```

Comment:
Oh, do we need to rebuild the `dist` files or actually, should we `.gitignore` the dist folder?

Reply:
I'm not sure how to deal with `dist`. It will be updated by semantic-release upon publish and need to be present when released for the action to work.

Maybe we could add a `ncc build` hook 🤔